packages/editor-ui/src/stores/cloudPlan.store.ts

```diff
@@ -163,6 +168,17 @@ export const useCloudPlanStore = defineStore(STORES.CLOUD_PLAN, () => {
 		window.location.href = `https://${adminPanelHost}/login?code=${code}`;
 	};
 
+	const fetchLeadEnrichmentTemplates = async () => {
+		try {
+			const leadEnrichmentTemplates = await getLeadEnrichmentTemplates(rootStore.getRestApiContext);
```

Comment:
we designed this feature to be generic.. ideally it would not be just for lead enrichment.. we want to be able to test other customizations.. 
seems like a lot of this code expects lead enrichment only.. please double check with @RicardoE105 .. the api should be returning the interest